x/wasm-storage/keeper/abci.go

```diff
@@ -1,18 +1,48 @@
 package keeper
 
 import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+
 	"cosmossdk.io/collections"
 	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/sedaprotocol/seda-wasm-vm/tallyvm"
 )
 
+type Request struct {
```

Comment:
Oh, Also realized some of this stuff is now hex or base 64 strings in the JSON instead of byte arrays/slices.

https://github.com/sedaprotocol/seda-common-rs/blob/main/src/msgs/data_requests/types.rs#L44

Reply:
fixed